libs/utils/Cargo.toml

```diff
@@ -35,6 +36,7 @@ strum_macros.workspace = true
 url.workspace = true
 uuid = { version = "1.2", features = ["v4", "serde"] }
 
+pq_proto.workspace = true
```

Comment:
So far, `pq_proto` was used in binary crates only and a `libs/postgres_backend` library crate.
`utils` is relatively bloated already, and there's been slow talks about breaking it up into pieces.

Should we put `pageserver_feedback.rs` into `postgres_backend` instead?

Reply:
Yeah, I'm also not a fan of that. But I'm not sure what would be better here. I tried moving pageserver_feedback.rs into postgres_backend, but the former depends on Lsn aka lsn.rs, and making pq_proto depend on utils sounds even worse. We could try separting 300 hundred lines Lsn into its own crate, but then lsn.rs depends on seqwait.rs. We could separate it as well, but is it worth it?

Reply:
Not sure: the API entities like these would be good to move away eventually, but it's not really the task of this PR indeed.